contrib/devtools/security-check.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python2
```

Comment:
There are 5 files using python2, only security-check.py and symbol-check.py (notably not test-security-check.py) have been changed in #7723. Accordingly I've changed only the same two files.

contrib/devtools/security-check.py:#!/usr/bin/env python2
contrib/devtools/symbol-check.py:#!/usr/bin/env python2
contrib/devtools/test-security-check.py:#!/usr/bin/env python2
contrib/zmq/zmq_sub.py:#!/usr/bin/env python2
share/rpcuser/rpcuser.py:#!/usr/bin/env python2 

I'm happy to abandon this PR in favour of incorporating the changes directly in #7723 if it simplifies things.